### DISCUSSION TITLE
Redirect from previous org owner

### DISCUSSION THREAD

**Discussion by benjdlambert** [community/community#28173] [Other Feature Feedback, Questions, & Ideas]
Hey :wave:

I'm one of the maintainers of https://github.com/backstage/backstage, and we have a weird problem that I think is down to the previous owner of the Backstage Org name. I believe that we were lucky enough to get the previous owners of the Organization to rename their org to something else, but it looks like now any package that doens't exist in our repo, will try to redirect to the old owners, which is a little unexpected when some of our users try to guess the package name that might exist in our repo and get redirected to another owner.

I've worked around this issue for now, by creating a private repo under the namespace to stop the redirect, but wondered if it's possible to remove all the redirects to a different organization?

The package in question is https://github.com/backstage/backstage-cli which redirects to https://github.com/apihub/apihub-cli

**Comment by jfullstackdev** [comment]
maybe here but not very sure of that 

https://support.github.com/request

yes, renaming Org and even our username are very tricky and there are unintended side-effects, maybe that is similar to the situation that when a username is released and is taken by another user, then if he / she created a repo that has the same repo name from the past user, it will not redirect anymore to the old project but the latest one

**Comment by airtower-luna** [comment]
> which is a little unexpected when some of our users try to guess the package name that might exist in our repo and get redirected to another owner

This is [documented behavior](https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization#changes-that-occur-automatically) when an organization (or user account) is renamed:

> GitHub automatically redirects references to your repositories. Web links to your organization's existing repositories will continue to work.

It is meant to make it easier for users who might still have old links. Of course they are strongly encouraged to update, because as you noticed any repositories you create can break redirects.
